CMSgov/eAPD
# Tbolt/3676 Admin Check

Resolves #3676 
Resolves #3675 
Resolves #3544

### Description

Adds initial react components for the admin check panel. Some additional notes-
* The data is mocked and not real. Everything in the “collapsed” is hardcoded HTML.
* Still need to come up with semantics for the custom parts of the panel. (Ie. What is the Incomplete Required Fields? A heading? How is the # tied to it?)


### Signifcant changes or possible side effects
* There shouldn't be any impact to existing functionality or new side effects introduced

### Automated test cases written
* Some test cases were created to test the basic pieces around the panel:
  * Opening and closing
  * Showing the completed state
  * Collapsing/Expanding the mini check
* More testing will be completed in subsequent stories
* Cypress tests are going to be a separate ticket

### Steps to manually verify this change

1. Login as a user who can view/edit APDs
2. Navigate to to APD export and submit page and click Run Administrative Check
3. Verify Admin Panel appears and matches criteria in original ticket

### This pull request is ready to review when


### This pull request can be merged when
